### ISSUE TITLE
error openmultiboot for openPLi image 

### ISSUE THREAD

**Issue by Dima73** [oe-alliance/openmultiboot#23]
As you know, I did support openmultiboot for openPLi.
I have recently received a lot of complaints that openmultiboot stopped working correctly.
Example:
flash image openpli 4.0
folder /open-multiboot/openatv-5.3-formuler1-20170110_usb
next boot --> openatv-5.3 --> reboot reciever
boot menu only one entry --> openpli 4.0(flash)
not entry openatv-5.3-formuler1-20170110_usb!

this log:
`=~=~=~=~=~=~=~=~=~=~=~= PuTTY log 2017.01.10 20:30:53 =~=~=~=~=~=~=~=~=~=~=~=

openpli 4 formuler1

formuler1 login: root
root@formuler1:~# open_multiboot
open-multiboot - debug: omb_utils_init_system            : mount /proc
open-multiboot - debug: omb_utils_init_system            : mount /sys
open-multiboot - debug: omb_utils_init_system            : mount /media
open-multiboot - debug: omb_utils_init_system            : run volatile media
open-multiboot - debug: omb_utils_init_system            : run mdev
open-multiboot - debug: omb_utils_find_and_mount         : check device /dev/sda1
open-multiboot - debug: omb_utils_find_and_mount         : found data on device /dev/sda1
open-multiboot - debug: omb_utils_get_images             : discover images
open-multiboot - debug: omb_branding_is_compatible       : processing /omb/open-multiboot/openatv-5.3-formuler1-20170110_usb
open-multiboot - debug: omb_branding_is_compatible       : brand_oem = 
open-multiboot - debug: omb_branding_is_compatible       : box_type_inflash = 
open-multiboot - debug: omb_branding_is_compatible       : box_type = formuler1
open-multiboot - debug: omb_branding_is_compatible       : box_type_inflash: != box_type:formuler1
open-multiboot - debug: omb_utils_get_images             : skipping image /omb/open-multiboot/openatv-5.3-formuler1-20170110_usb
open-multiboot - debug: omb_utils_prepare_destination    : prepare destination
open-multiboot - debug: omb_utils_read_int               : selected 0
open-multiboot - debug: main                             : preparing environment...
open-multiboot - debug: main                             : loading modules...
open-multiboot - debug: omb_utils_load_modules           : load modules
Loading modules: formuler1_1 formuler1_2 formuler1_3 formuler1_4 
/dev/dvb/adapter0/video0: Device or resource busy
open-multiboot - debug: omb_utils_backup_kernel          : backup kernel for image 'flash'
libmtd: error!: "/dev/" is not a character device
nanddump: error!: mtd_get_dev_info failed
open-multiboot - debug: main                             : menu enabled
open-multiboot - debug: main                             : boxmodel: 
open-multiboot - debug: omb_open_framebuffer             : the framebuffer device was opened successfully
open-multiboot - debug: omb_read_screen_info             : current mode is 720x576, 32bpp, stride 2880
open-multiboot - debug: omb_read_screen_info             : current mode is 1920x1080, 32bpp, stride 7680
open-multiboot - debug: omb_map_framebuffer              : the framebuffer device was mapped to memory successfully
open-multiboot - debug: omb_make_palette                 , set color palette disabled: FIXME !!
open-multiboot - debug: omb_set_manual_blit              : set manual blit
open-multiboot - debug: omb_input_open                   : input device opened
Usage: init.sysvinit {-e VAR[=VAL] | [-t SECONDS] {0|1|2|3|4|5|6|S|s|Q|q|A|a|B|b|C|c|U|u}}`

**Comment by dpeddi** [comment]
please give a try to "switch_stick_safe" branch

**Comment by Dima73** [comment]
for openpli not branding support.
maybe it is possible to make the setting (branding support-yes/no)?
You do not have to binary formuler1 / switch_stick_safe" branch?

**Comment by dpeddi** [comment]
- maybe it is possible to make the setting (branding support-yes/no)?
Maybe yes.. feel free to propose a pull request. Probably I will include your proposal within some weeks...

- You do not have to binary formuler1 / switch_stick_safe" branch?
I haven't understood..

**Comment by Dima73** [comment]
# Maybe yes.. feel free to propose a pull request. Probably I will include your proposal within some weeks...

**Comment by dpeddi** [comment]
no, I don't have that stb... but since that binary use branding don't waste your time

**Comment by arn354** [comment]
We have to options for 3 options for openpli
A) Pli builds boxbranding and adds boxbranding to depends of openmultiboot (smallbinary, does not harm in anyway if installed - just eases up live).
B) we create a branch Pli with commithash before last changes which require boxbranding and Pli builds from there
C) Pli forks it before last changes and builds from fork

**Comment by Dima73** [comment]
Let's make openmultiboot is compatible with all image.
The simplest and best solution:
maybe it is possible to make the setting (branding support-yes/no)?

**Comment by dpeddi** [comment]
Easy, introduce support for boxbranding in your image ;-)
